src/twisted/_checkrequirements.py

```diff
@@ -0,0 +1,14 @@
+# -*- test-case-name: twisted -*-
+
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+def _checkRequirements():
+    # Don't allow the user to run a version of Python we don't support.
```

Comment:
I know that this is a copy/paste, but maybe take the oportunity to clean this code a bit and for example convert this comment into a docstring.

And why the local import?

Reply:
OK, I made it a docstring